### PR TITLE
chore: release v1.5.0

### PR DESCRIPTION
**Pull request by eunjae-lee**
## Release Summary
- Version change: `v1.4.0` → `v1.5.0`
- Merge: `releases/v1.5.0` → `master`
- [Compare the changes between the versions](https://github.com/algolia/search-insights.js/compare/v1.4.0...releases/v1.5.0)
> :warning: When merging this pull request, you need to **_"Squash and merge"_** and make sure the title starts with `chore: release v1.5.0`.
> After that, a commit `chore: release v1.5.0` will be added and `shipjs trigger` will be able to trigger the release based on the commit.
> Fore more information, please refer to the mergeStrategy section of the [guide](https://github.com/algolia/shipjs/blob/master/GUIDE.md#mergestrategy).
> Squash and merge

---
This is going to be published by the following command:
```
yarn publish --no-git-tag-version --non-interactive --tag latest
```
---
_This pull request is automatically generated by [Ship.js](https://github.com/algolia/shipjs)_

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+# [1.5.0](https://github.com/algolia/search-insights.js/compare/v1.4.0...v1.5.0) (2020-07-23)
+
+
+### Features
+
+* **browser:** expose createInsightsClient ([#213](https://github.com/algolia/search-insights.js/issues/213)) ([7880205](https://github.com/algolia/search-insights.js/commit/788020598064bc5f9c71ba0b1cf5a20109195fb8))
+* **get:** add _get function ([#216](https://github.com/algolia/search-insights.js/issues/216)) ([3173c9b](https://github.com/algolia/search-insights.js/commit/3173c9bcefb15c9c9b166636dabef6945e555f6f))
```
Comment (Haroenv) [inline CHANGELOG.md:7]: This part might not be so clear, but probably no big deal

Comment (eunjae-lee) [reply CHANGELOG.md]: updated :)

**Comment by francoischalifour** [comment]
Previous PRs were merged quickly, maybe too quickly to get feedback before releasing.

**Comment by eunjae-lee** [comment]
@francoischalifour You're right. I'll slow down a bit next time.

**Pull request merged**